GrpcClient/ShiftGrpcRepository.cs

```diff
@@ -0,0 +1,257 @@
+﻿using DTOs.Shift;
+using Entities;
+using Google.Protobuf.WellKnownTypes;
+using Grpc.Core;
+using Grpc.Net.Client;
+using RepositoryContracts;
+
+namespace GrpcClient;
+
+public class ShiftGrpcRepository : IShiftRepository
+{
+    private string _grpcAddress { get; set; }
+
+    public ShiftGrpcRepository()
+    {
+        _grpcAddress = "http://192.168.140.143:50051";
+    }
+
+    public async Task<Entities.Shift> AddAsync(Entities.Shift shift)
+    {
+        
+        using var channel = GrpcChannel.ForAddress(_grpcAddress);
+        
+        var client = new Shift.ShiftClient(channel);
+        
+        var reply = await client.AddSingleShiftAsync(new NewShiftDTO
+        {
+            TypeOfShift = shift.TypeOfShift,
+            Location = shift.Location,
+            ShiftStatus = shift.ShiftStatus,
+            Description = shift.Description,
+            StartDateTime = new DateTimeOffset(shift.StartDateTime).ToUnixTimeMilliseconds(),
+            EndDateTime = new DateTimeOffset(shift.EndDateTime).ToUnixTimeMilliseconds(),
+            
+        });
+        
+        Entities.Shift shiftRecieved = grpcShiftObject(reply);
```

Comment:
What is grpcShiftObject?

Reply:
converter that marius made for grpc shift -> (<-) csharp shift

Reply:
@Craym fix plz? and other converters